### PR TITLE
xfce-base/xfce4-settings: fix undefined references

### PR DESCRIPTION
**Pull request by orbea**
Closes: https://bugs.gentoo.org/913864
Upstream-Issue: https://gitlab.xfce.org/xfce/xfce4-settings/-/issues/494
Upstream-Commit: https://gitlab.xfce.org/xfce/xfce4-settings/-/commit/f13b9ff1e5361439eafb9cffafd4d23fa6b2cb9e

### CODE REVIEW

File: xfce-base/xfce4-settings/files/xfce4-settings-4.19.1-gmodule-libs.patch [hunk 1]
```diff
@@ -0,0 +1,35 @@
+https://bugs.gentoo.org/913864
+https://gitlab.xfce.org/xfce/xfce4-settings/-/issues/494
```
Comment (mgorny) [inline xfce-base/xfce4-settings/files/xfce4-settings-4.19.1-gmodule-libs.patch:2]: Put that above the filename in ebuild, so people don't have to open a second file to check.

File: xfce-base/xfce4-settings/xfce4-settings-4.19.1.ebuild [hunk 2]
```diff
@@ -63,6 +63,15 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.19.1-gmodule-libs.patch #913864
```
Comment (mgorny) [inline xfce-base/xfce4-settings/xfce4-settings-4.19.1.ebuild:67]: Don't use bare bug numbers, or trailing bash comments.

**Review by mgorny — CHANGES_REQUESTED** [review]
We don't have any of these `Upstream*` tags, we link upstream bugs via `Bug`.

File: xfce-base/xfce4-settings/files/xfce4-settings-4.19.1-gmodule-libs.patch (hunk 1 above)
Comment (orbea) [reply xfce-base/xfce4-settings/files/xfce4-settings-4.19.1-gmodule-libs.patch:2]: Done.

File: xfce-base/xfce4-settings/xfce4-settings-4.19.1.ebuild (hunk 2 above)
Comment (orbea) [reply xfce-base/xfce4-settings/xfce4-settings-4.19.1.ebuild]: Done.

Although this is a very common pattern which I copied from other ebuilds when I first started trying to submit patches to Gentoo and have never been told as being wrong before.

File: xfce-base/xfce4-settings/xfce4-settings-4.19.1.ebuild [hunk 3]
```diff
@@ -63,6 +63,17 @@ BDEPEND="
...
 
+PATCHES=(
+	# https://bugs.gentoo.org/913864
+	# https://gitlab.xfce.org/xfce/xfce4-settings/-/issues/494
+	"${FILESDIR}"/${PN}-4.19.1-gmodule-libs.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
```
Comment (mgorny) [inline xfce-base/xfce4-settings/xfce4-settings-4.19.1.ebuild:74]: Actually, I think `eautomake` should be sufficient, shouldn't it?

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**